### DISCUSSION TITLE
Unable to get the title of the option in the API, instead receiving the value..

### DISCUSSION THREAD

**Discussion by ivanhirskyy** [directus/directus#16953] [Archive]
### Describe the Bug

Hey, 
I'm having a problem with dropdowns, which basically is that I cant receive the text of the option in the API response.

![image](https://user-images.githubusercontent.com/69305066/209935547-34ba02e6-0942-4ddc-b015-a8ca0eff722b.png)

The value that I want is the left one
![image](https://user-images.githubusercontent.com/69305066/209935647-7ee332cf-7ce0-49b9-b044-cebe29e6e866.png)

How come I don't have the option to receive the text ? Why would I want to show a user the value and not the text ?
![image](https://user-images.githubusercontent.com/69305066/209935870-196b8534-6e1b-420f-b999-7b5a17aa6cea.png)

I would really use some help :)

### To Reproduce

Create a table, with a dropdown field, fill the options, than do a API request for those values

### What version of Directus are you using?

9.22.1

### What version of Node.js are you using?

18.12.1

### What database are you using?

Its the cloud version

### What browser are you using?

Chrome

### How are you deploying Directus?

Its the cloud version

**Comment by br41nslug** [comment]
Dropdown labels are saved on the field as interface options and are not used for the API. To translate the values to the display labels you'll have to fetch these settings and make the translation on the frontend (similar as the the App does). 
https://docs.directus.io/reference/system/fields.html#retrieve-a-field

**Reply by ivanhirskyy** [reply]
So basically I need to relate the values with some expressions on the frontend?

**Reply by br41nslug** [reply]
To get them translated from value to title yes. Another approach would be to use the same values for `value` and `title` to prevent any translation steps.

**Reply by ivanhirskyy** [reply]
The second aproach is better, because if I would translate from value to title, I would need to do it everytime I had a new option.
And also I just need 1 language
